src/Compilers/Core/Portable/Operations/OperationKind.cs

```diff
@@ -183,6 +183,7 @@ public enum OperationKind
         DeclarationPattern = 0x56,
 
         FlowCapture = 0x57,
```

Comment:
Consider adding a doc comment on the same lines as other operation kinds.

Reply:
>Consider adding a doc comment on the same lines as other operation kinds.

Will add in the next PR

---
In reply to: [163292616](https://github.com/dotnet/roslyn/pull/24358#discussion_r163292616) [](ancestors = 163292616)